app/cognito/utils/user.py

```diff
@@ -0,0 +1,58 @@
+from logging import getLogger
+
+from cognito.utils.client import Client
+
+logger = getLogger(__name__)
+
+
+# TODO: Replace me with the actual user model
+class User:
+    id: int
+    email: str
+
+
+def add_user(user: User) -> None:
+    """ Add the given user to cognito.
+
+    Update the user, if he already exists.
+    """
+
+    client = Client()
+    username = str(user.id)
+    existing = client.get_user(username)
+    if existing is not None:
+        logger.warning("User %s already exists, updating", username)
+        client.update_user(username, user.email)
+    else:
+        client.create_user(username, user.email)
+    logger.info("User %s created", username)
```

Comment:
This will print "created" regardless of if the user was updated or created. Is that intended?

Reply:
Yes, that was intended, but not sure if it's a good idea. Can change for more clarity.